app/legacy_lib/insert_card.rb

```diff
@@ -20,8 +20,8 @@ module InsertCard
     def self.with_stripe(card_data, stripe_account_id=nil, event_id=nil, current_user = nil)
 
     begin
-      ParamValidation.new(card_data.to_deprecated_h.merge({event_id: event_id}), {
```

Comment:
Why was this removed?

Reply:
I am not sure. 🤔  I will replace the removal.